src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/SetupAssistant.kt

```diff
@@ -259,10 +270,13 @@ class SetupAssistant(
     attended: Attended? = null,
     attendanceInfo: String? = null,
     behaviour: String? = null,
-    notifyPPOfBehaviour: Boolean? = null
+    notifyPPOfBehaviour: Boolean? = null,
+    appointmentDeliveryType: AppointmentDeliveryType? = null,
+    appointmentDeliveryAddressLines: List<String>? = null,
```

Comment:
is this list of strings just used in the tests? the data we get in the API call is more structured than this right?